vms/platformvm/reward/calculator.go

```diff
@@ -67,3 +69,16 @@ func (c *calculator) Calculate(stakedDuration time.Duration, stakedAmount, curre
 
 	return finalReward
 }
+
+func Split(totalAmount uint64, shares uint32) (uint64, uint64) {
```

Comment:
Can we document this? (specifically the return values and how `shares` are expected to be defined).

Reply:
Done